examples/pipeline/sample.tfvars

```diff
@@ -56,7 +55,7 @@ organization = {
     sandbox = {
       account_name  = "primeharbor-kickstart-sandbox"
       account_email = "aws+kickstart-sandbox@primeharbor.com"
-      parent_ou_id  = "ou-yyyy-yyyyyyyy"
+      parent_ou_id  = "Sandbox"
```

Comment:
This (and the related changes below) is another important update, which aligns the docs with improvements made in d13c8413c159c61a78755917e4adf893fb15aad3 and c35332ef58c73be47f1286fab3a3179218691e94